src/nft/components/collection/CollectionStats.css.ts

```diff
@@ -67,6 +67,23 @@ export const description = style([
   },
 ])
 
+globalStyle(`${description} a[href]`, {
```

Comment:
We should try to avoid using global styles to avoid unintended consequences. You can add the selectors to the `description` style as shown here: https://vanilla-extract.style/documentation/styling/#selectors

Reply:
if you scroll down on that linked documentation page Charlie it says: 
"If you need to globally target child nodes within the current element (e.g. '& a[href]'), you should use [globalStyle](https://vanilla-extract.style/documentation/global-api/global-style) instead." 

I tried using selectors with a[href] previously and it doesn't work (no styling changes are applied). seems to be an exception case where we _need_ to use globalStyle.